app/lib/widgets/cards/productCard/shared/price.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flex_ui/utils/extensions.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+import 'package:intl/intl.dart';
+
+enum Variant {
+  standard,
+  savings, // Defaults to text decoration linethrough and color 'disabled'
+}
+
+class Price extends StatelessWidget {
+  const Price({
+    super.key,
+    required this.price,
+    this.formatterCallback,
+    this.variant = Variant.standard,
+    this.textStyle,
+  });
+  final double price;
+  final String Function(double)? formatterCallback;
+  final Variant? variant;
```

Comment:
Let's name `PriceVariant` and we are setting the default in the constructor so it's non-nullable here

Reply:
Coming from the google chat - thoughts on variant naming
we want to be more utilitarian on variant titles given this core widget isn't directly tied to any sale display logic? 

Ie ``PriceVariant.strikeThrough`` vs current ``PriceVariant.savings``

@stephenrichter @romgrm

Reply:
I'm good with strikethrough :)